react/components/MiniCartItem.js

```diff
@@ -95,7 +95,7 @@ export default class MiniCartItem extends Component {
       <div className={`${minicart.item} h4 relative w-100`}>
         <Link
           className="pointer link black-90"
-          page={'store/product'}
+          page={'store.product'}
```

Comment:
It was buggy? If so, please add it in the CHANGELOG, so it would be easier for other developers to check where some bug was corrected